ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resourcepool/cucushift-installer-check-vsphere-customized-resourcepool-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+unset SSL_CERT_FILE
+unset GOVC_TLS_CA_CERTS
+check_result=0
+
+machineset_name=$(oc get machineset -n openshift-machine-api --no-headers | head -n1 | awk '{print $1}')
+rsp_config=$(yq-go r ${INSTALL_CONFIG} 'platform.vsphere.failureDomains[*].topology.resourcePool')
```

Comment:
It's better to check if `rsp_config` is empty, and skip the check and exit 0 when it is empty.

Reply:
ok.